docker-images/kafka/scripts/kafka_run.sh

```diff
@@ -25,8 +25,35 @@ fi
 rm /var/opt/kafka/kafka-ready /var/opt/kafka/zk-connected 2> /dev/null
 export KAFKA_OPTS="-javaagent:$(ls $KAFKA_HOME/libs/kafka-agent*.jar)=/var/opt/kafka/kafka-ready:/var/opt/kafka/zk-connected"
 
+if [ "$KAFKA_JMX_ENABLED" = "true" ]; then
+  KAFKA_JMX_OPTS="-Dcom.sun.management.jmxremote.port=9999 -Dcom.sun.management.jmxremote.rmi.port=9999 -Dcom.sun.management.jmxremote=true -Djava.rmi.server.hostname=$(hostname -i) -Djava.net.preferIPv4Stack=true"
+
+  if [ -n "$KAFKA_JMX_USERNAME" ]; then
+   # Secure JMX port on 9999 with username and password
+    ACCESS_FILE="/tmp/access.file"
+    PASSWORD_FILE="/tmp/password.file"
+
+  cat << EOF > "${ACCESS_FILE}"
+${KAFKA_USERNAME} readonly
+EOF
+
+  cat << EOF > "${PASSWORD_FILE}"
+$KAFKA_USERNAME $KAFKA_PASSWORD
+EOF
+  chmod 400 "${PASSWORD_FILE}"
```

Comment:
You don't have to change this, but personally I much prefer the mnemonic form `chmod u=r,go-rwx` which is, I think, less cryptic than the octal.